Resources/Prototypes/_DV/Entities/Structures/Storage/Closets/Lockers/wall_lockers.yml

```diff
@@ -0,0 +1,236 @@
+- type: entity
+  parent: BaseWallCloset
+  id: BaseWallClosetDeltaV
+  components:
+  - type: Sprite
+    sprite: _DV/Structures/Storage/wall_locker.rsi
+
+- type: entity
+  parent: BaseWallLocker
+  id: BaseWallLockerDeltaV
+  components:
+  - type: Sprite
+    sprite: _DV/Structures/Storage/wall_locker.rsi
+
+- type: entity
+  parent: BaseWallClosetDeltaV
+  id: WallClosetTools
+  name: tool wall closet
+  components:
+    - type: Appearance
+    - type: EntityStorageVisuals
+      stateBaseClosed: engi_generic
+      stateDoorClosed: engi_tools_door
+      stateDoorOpen: engi_generic_open
```

Comment:
remove appearance and unindent the other component by 2 spaces, for all of these